llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -117,6 +117,7 @@ class VPBlockBase {
     Predecessors.erase(Pos);
   }
 
+public:
```

Comment:
Hopefully this low-level API can remain private, possibly with an addition to the high-level public API, see below.

Reply:
Done thanks!